Source/JavaScriptCore/wasm/js/JSWebAssemblyArray.h

```diff
@@ -56,8 +56,11 @@ class JSWebAssemblyArray final : public WebAssemblyGCObjectBase {
 
     static Structure* createStructure(VM&, JSGlobalObject*, JSValue);
 
-    static JSWebAssemblyArray* create(VM& vm, Structure* structure, Wasm::FieldType elementType, size_t size, RefPtr<const Wasm::RTT>&& rtt)
+    static JSWebAssemblyArray* tryCreate(VM& vm, Structure* structure, Wasm::FieldType elementType, size_t size, RefPtr<const Wasm::RTT>&& rtt)
     {
+        // We have no good way to test for a failed allocation in FixedVector, so we open-code the test here.
```

Comment:
Why can't we just fix FixedVector?

Reply:
Allowing FixedVector allocation to fail would be a more far-reaching change in the `webkitglib/2.48` branch. Ideally, we'd go in the direction of https://commits.webkit.org/291760@main and fix `allocationSize` like ToT.